src/app/app-routing.module.ts

```diff
@@ -15,7 +17,8 @@ const routes: Routes = [
 	{ path: "forget", component: ForgetComponent },
 	{ path: "reset/:id", component: ResetComponent },
 	{ path: "pricing", component: PricingComponent },
-	{ path: "developer", component: DeveloperComponent }
+	{ path: "developer", component: DeveloperComponent, canActivate:[AuthGuard], data:{roles:['ADMIN']} },
+	{ path: 'forbidden', component: ForbiddenComponent }
```

Comment:
Need to add one more path 

{ path: "dashboard", component: MainDashboardComponent, canActivate:[AuthGuard], data:{roles:['EMPLOYER']} },